### ISSUE TITLE
Does not install @types at proper version.

### ISSUE THREAD

**Issue by JoshMcCullough** [T-Specht/ts-typie#3]
Thanks for this tool! 

I noticed that it seems to install the latest/current types for a package rather than the @types at the same version of the installed package.

When you run `ts-typie`, it should not only install missing @types (at the proper version), but also make sure that currently installed @types match the currently installed version of the related package.

**Comment by T-Specht** [comment]
You are right, installing the types corresponding with the version of the package would make sense. Since I am quite busy at the moment, implementing this will not be my number one priority, but you are more than welcome to file a pull request.

**Comment by Rycochet** [comment]
The only issue with this is that the **@types** version number might not exactly match the installed version number - there's at least a couple that I use (can't remember offhand which or I'd link) that use different version styles. More common is a suffix, but that's by no means universal.

As a solution to this, maybe add a `--check` option that does all the checking if they exist, but doesn't actually install anything - a second `--quiet` option could hide all output lines that are Skipped...

**Comment by sou-long** [comment]
Not just a couple of. Lots of type packages either fall behind for various reasons or ignore original versioning within major releases or completely. There's no types / releases version correspondence built into typescript+npm ways, except for typescript version itself. It's an afterthought by design. Packages that do care about exact dts versions ought to contain them.
